docs/reference/configuration/vehicles.mdx

```diff
@@ -123,9 +123,15 @@ Standardwerte die bei Erkennung des Fahrzeuges angewendet werden sollen:
 - [`mode`](loadpoints#mode)
 - [`minCurrent`](loadpoints#mincurrent)
 - [`maxCurrent`](loadpoints#maxcurrent)
+- [`priority`](https://docs.evcc.io/docs/reference/configuration/loadpoints/#priority)
 - `minSoc` es wird sofort mit höchstmöglicher Leistung bis zum angegeben Füllstand geladen, wenn der Modus nicht auf `off` steht
 - `targetSoc` bei Erreichen dieses Füllstandes wird die Ladung gestoppt
-- [`priority`](https://docs.evcc.io/docs/reference/configuration/loadpoints/#priority)
+
+:::info
+
+Werden die Werte für `minSoc` und `targetSoc` im WebUI geändert, werden diese dauerhaft beim Fahrzeug gespeichert. Auf die EInstellungen in der evcc.yaml wird dann nur noch bei einem Neustart zurückgegriffen.
```

Comment:
Ich würd das sogar noch härter formulieren. Eigentlich sollten diese Settings nicht mehr in der evcc.yaml gepflegt werden. Wir könnten die hier als `deprecated` markieren.

Reply:
> OK, aber ist das falsch?

Nein, ist nicht falsch. Aber ich sehe aktuell keinen Grund warum ein Nutzer noch den Config-Wert in der yaml nutzen sollte - außer Abwärtskompatiblität.

> Hier müsste die Doku jetzt bebildert werden.

Einige Funktionen versucht die UI ja schon selbst zu erklären. MinSoC hat bspw. in der UI einen Erkärungstext bekommen. Bei der Ladeplanung machts (hoffentlich) das Diagramm.

Trotzdem machts natürlich Sinn auch ne ausführlichere Erklärung in der Dokumentation zu haben. Ich könnte mir vorstellen einen neuen Navigationspunkt "Funktionen" in die Hauptnavi aufzunehmen. Darunter würde ich sowas wie "Ladelimit", "Mindestladung", "Ladeplanung", "Ladevorgänge", "Batterie", "Optimiertes Netzladen", ... sehen. Ich würd allen Funktionen eine eigene Seite verpassen die dann ggf. auch auf die Referenz verlinkt. Einiges von diesen Themen ist aktuell auch schon in "Tipps & FAQ" abgebildet. Das sollten wir migrieren.

Bilder hochladen geht auch, wobei die natürlich aufwändiger nachzupflegen sind. Aber eigentlich spricht da sonst nichts gegen.

Wie siehst du das? Ich kann so eine Struktur mit Beispielinhalt (und Screenshot) auch gerne mal anlegen.

Reply:
>Wie siehst du das? Ich kann so eine Struktur mit Beispielinhalt (und Screenshot) auch gerne mal anlegen.

Das wäre nicht schlecht. Gucken und anfassen ist mit lieber, als nur gedanklich vorstellen ;)

Reply:
schau mal  https://github.com/evcc-io/docs/issues/396#issuecomment-1591621871

Reply:
@naltatis 
>Ich würd das sogar noch härter formulieren. Eigentlich sollten diese Settings nicht mehr in der evcc.yaml gepflegt werden. Wir könnten die hier als deprecated markieren.

sollte jetzt okay sein